assets/filter_repos.js

```diff
@@ -20,7 +20,7 @@
                 for (let j = 0; j < repoHooks.length; j += 1) {
                     const repoHook = repoHooks[j];
                     const hookId = repoHook.dataset.id.toLowerCase();
-                    const hookTypes = repoHook.dataset.types;
+                    const hookTypes = repoHook.dataset.types.split(', ');
 
                     if (hookId.includes(id) && hookTypes.includes(type)) {
```

Comment:
Is it enough to change this line like this?
```diff
- if (hookId.includes(id) && hookTypes.includes(type)) {
+ if (hookId.includes(id) && (type === '' || hookTypes.includes(type))) {
```

Reply:
yeah that looks good -- want to send a PR?

Reply:
Done. https://github.com/pre-commit/pre-commit.com/pull/522